src/main/diagnostics/descriptorActionProvider.ts

```diff
@@ -1,15 +1,17 @@
 import * as vscode from 'vscode';
 
-// import { CveTreeNode } from '../treeDataProviders/issuesTree/descriptorTree/cveTreeNode';
 import { DescriptorTreeNode } from '../treeDataProviders/issuesTree/descriptorTree/descriptorTreeNode';
-// import { LicenseIssueTreeNode } from '../treeDataProviders/issuesTree/descriptorTree/licenseIssueTreeNode';
-// import { LicenseIssueTreeNode } from '../treeDataProviders/issuesTree/descriptorTree/licenseIssueTreeNode';
 import { FileTreeNode } from '../treeDataProviders/issuesTree/fileTreeNode';
+import { IssueTreeNode } from '../treeDataProviders/issuesTree/issueTreeNode';
 import { DescriptorUtils } from '../treeDataProviders/utils/descriptorUtils';
 import { PackageType } from '../types/projectType';
 import { Severity, SeverityUtils } from '../types/severity';
 import { AbstractFileActionProvider } from './abstractFileActionProvider';
 
+class InfoPair {
```

Comment:
What information does this class hold and who is paired with whom? Is it necessary to create another class here?

Reply:
I change the naming to fir better: DirectDependencyInfo
we need this because we are putting the diagnostics on a direct dependency while we only show and save information about the specific dependency with the issue, in this class we aggregate the issues for the direct dependency.
this is an inner class that is not exposed and used to hold the data while we aggregate from all the issues